### ISSUE TITLE
Add a while loop that iterates over newlines, vs any whitespace

### ISSUE THREAD

**Issue by kvz** [kvz/bash3boilerplate#46]
e.g.

``` bash
IFS=$'\n'
for line in $(cat many-lines.txt); do
  echo "${line}"
done
unset IFS
```

**Comment by zbeekman** [comment]
I'm not sure I understand where/why you want to add this. Just as an example, or to enhance the functionality of some part of b3bp?

**Comment by kvz** [comment]
Sorry that wasn't clear! I was thinking to put it at the bottom where we also fire the log functions, as an example, and easy to copy-paste bit right in the main template. What do you think?

**Comment by mstreuhofer** [comment]
seems to me that this issue is fixed with #57

**Comment by kvz** [comment]
I guess people can take this type of loop straight out of `__b3bp_log` now, yes. Will close it this issue then :+1:
